crates/ibc/Cargo.toml

```diff
@@ -46,7 +46,7 @@ schema = ["dep:schemars", "serde", "std"]
 
 # This feature grants access to development-time mocking libraries, such as `MockContext` or `MockHeader`.
 # Depends on the `testgen` suite for generating Tendermint light blocks.
-mocks = ["tendermint-testgen", "tendermint/clock", "parking_lot", "typed-builder"]
+mocks = ["tendermint-testgen", "tendermint/clock", "parking_lot", "typed-builder", "std"]
```

Comment:
A nit: We can drop the `tendermint/clock` for `mocks` as it already lives in `std`